packages/dai-plugin-governance/src/ChiefService.js

```diff
@@ -223,13 +223,13 @@ export default class ChiefService extends LocalService {
   getNumDeposits(address) {
     return this._chiefContract()
       .deposits(address)
-      .then(MKR.wei);
+      .then(n => MKR.wei(n._hex));
```

Comment:
Since all our contracts are now created with ethers 5, the returned number value is of BN.js bignumber type (not bignumber.js). Passing a BN bignumber into `currency` does not work, so instead we now have to pass it's hex value (passing `.toString()` would also work, but I figure the hex value is less prone to a conversion error).

This is the first of many examples of converting an ethers returned BN type into a hex for `currency` to consume.